strapi/strapi
# [Custom fields] list custom fields

### What does it do?

- Lists the customFields on the attribute create modal

### How to test it?

- Open any content-type and click add attribute
- Click the custom tab, you should see two custom fields
- They should be sorted alphabetically by customField name
- Click the "How to create custom fields" link it should take you to the docs

**NOTE**
- You can click a customField option and it will open the form of the underlying data type, you can fill out the form and add it to the contentType, and save the contentType. It will save as the underlying data type in the schema.json. This is expected at this stage.
